olive/passes/onnx/gptq.py

```diff
@@ -0,0 +1,165 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from typing import Any, Dict
+
+import torch
+from onnxruntime import __version__ as ort_version
```

Comment:
As we target to remove the dependencies of onnxruntime, please move it to proper function.